spec/features/idv/steps/phone_step_spec.rb

```diff
@@ -92,13 +92,14 @@
     context 'resubmission after number failed verification' do
       it 'phone field is empty after invalid submission' do
         phone_field = find_field(t('two_factor_authentication.phone_label'))
-
         expect(phone_field.value).not_to be_empty
 
         click_idv_send_security_code
         click_on t('idv.failure.phone.warning.try_again_button')
 
         expect(page).to have_current_path(idv_phone_path)
+
+        phone_field = find_field(t('two_factor_authentication.phone_label'))
```

Comment:
What was going on here?

Reply:
Chromedriver was failing with an 'outdated element' explosion. Tracked down, and re-finding the field fixed it. No idea why it ever worked.